_includes/asg/aws-prereqs.md

```diff
@@ -0,0 +1,37 @@
+### AWS Requirements
+
+As Portworx needs to create and attach EBS volumes, it needs a few AWS permissions. Following is a sample policy describing which permissions Portworx needs:
```

Comment:
change to "it needs corresponding AWS permissions"

Reply:
done!